foundation/foundation-mda/src/main/resources/templates/deployment/argocd/v2/spark.infrastructure.yaml.vm

```diff
@@ -6,6 +6,15 @@ metadata:
   finalizers:
     - resources-finalizer.argocd.argoproj.io
 spec:
+  ignoreDifferences:
+    - group: ""
+      kind: ConfigMap
+      managedFieldsManagers:
+        - argocd-controller
+    - group: apps
+      kind: StatefulSet
+      managedFieldsManagers:
+        - argocd-controller
```

Comment:
I: ManagedFields are already there and injected fields are included, we just need to ignore diff on that specific manager https://argo-cd.readthedocs.io/en/stable/user-guide/diffing/#application-level-configuration